website/static/js/logTextParser.js

```diff
@@ -446,7 +446,10 @@ var LogPieces = {
                     'github_file_added', 'github_file_updated', 'box_file_added', 'box_file_updated', 'dropbox_file_added', 'dropbox_file_updated',
                     's3_file_added', 's3_file_updated', 'figshare_file_added', 'checked_in', 'checked_out'];
                     if (acceptableLinkedItems.indexOf(action) !== -1 && logObject.attributes.params.urls) {
-                         return logObject.attributes.params.urls.view;
+                        if (logObject.attributes.params.params_file){
```

Comment:
We get a lot of errors in Sentry where we try to deep access objects and they aren't complete before they're there. https://sentry.cos.io/sentry/osf-front-end/issues/276034/ is an example where we are trying `if (!ctrl.node.anonymous && item.embeds.user && item.embeds.user.data)` but item.embeds doesn't exist and so it gives a TypeError. You should use lodash.get for this, I think. https://github.com/CenterForOpenScience/osf.io/search?utf8=✓&q=lodash.get&type